app/demo-hifive-inventor/app.toml

```diff
@@ -6,25 +6,41 @@ stacksize = 896
 
 [kernel]
 name = "demo-hifive-inventor"
-requires = { flash = 16752, ram = 3232 }
+requires = { flash = 16784, ram = 3472 }
 features = []
 
 [tasks.jefe]
 name = "task-jefe"
 priority = 0
 max-sizes = { flash = 16384, ram = 2048 }
 start = true
-features = ["semihosting-riscv"]
+features = ["log-stringbuf"]
 stacksize = 1536
 
+[tasks.ext_int_ctrl]
+name = "drv-riscv-plic"
+priority = 1
+max-sizes = { flash = 4096, ram = 2048 }
+start = true
+features = ["log-stringbuf"]
+uses = ["plic"]
+interrupts = { "plic.irq" = 0x1 }
+
+[tasks.ext_int_ctrl.config]
+ints = [52]
```

Comment:
Another comment for the future. It would be good to see if we can extend the use of `.irq` to reference the interrupt numbers for peripherals. That will make it easier for app developers to just refer to that instead of putting magic numbers in here.

Reply:
Yeah that definitely would be preferable. Doing that is going to require some more contortion of the build system that I decided isn't worth handling for now. But, there would be very little in this patch that needs to be changed to make that work so there isn't a major cost to switching.

Reply:
Agreed. I just wanted to note it. You can revisit that later.